packages/odyssey-storybook/applitools.config.js

```diff
@@ -26,5 +26,5 @@ module.exports = {
   matchLevel: "Strict",
   parentBranchName,
   showStorybookOutput: true,
-  testConcurrency: 20,
+  testConcurrency: 10,
```

Comment:
what is the current issue? what is the optimal number ?

Reply:
The current issue is that we only have 20 concurrencies with Applitools in total and we have gone above it by 100% on multiple occasions recently. To avoid overcharging or forced upgrades at the renewal, we should monitor our usage and keep it below that 20 concurrency limit.  It's very unlikely that we have more than 2 Odyssey PRs running VRT at the same time, so 10 could be an optimal number in that case.

Reply:
Good to know !! ohh ok so the total thread is per account ?! hopefully only max 2 prs runs vrt at same time. 
Updated the jira ticket with this information.